src/Views/magic_link_form.php

```diff
@@ -9,37 +9,38 @@
         <div class="card-body">
             <h5 class="card-title mb-5"><?= lang('Auth.useMagicLink') ?></h5>
 
-                <?php if (session('error') !== null) : ?>
-                    <div class="alert alert-danger" role="alert"><?= session('error') ?></div>
-                <?php elseif (session('errors') !== null) : ?>
-                    <div class="alert alert-danger" role="alert">
-                        <?php if (is_array(session('errors'))) : ?>
-                            <?php foreach (session('errors') as $error) : ?>
-                                <?= $error ?>
-                                <br>
-                            <?php endforeach ?>
-                        <?php else : ?>
-                            <?= session('errors') ?>
-                        <?php endif ?>
-                    </div>
-                <?php endif ?>
+            <?php if (session('error') !== null) : ?>
+                <div class="alert alert-danger" role="alert"><?= session('error') ?></div>
+            <?php elseif (session('errors') !== null) : ?>
+                <div class="alert alert-danger" role="alert">
+                    <?php if (is_array(session('errors'))) : ?>
+                        <?php foreach (session('errors') as $error) : ?>
+                            <?= $error ?>
+                            <br>
+                        <?php endforeach ?>
+                    <?php else : ?>
+                        <?= session('errors') ?>
+                    <?php endif ?>
+                </div>
+            <?php endif ?>
 
             <form action="<?= url_to('magic-link') ?>" method="post">
                 <?= csrf_field() ?>
 
                 <!-- Email -->
                 <div class="mb-2">
-                    <input type="email" class="form-control" name="email" autocomplete="email" placeholder="<?= lang('Auth.email') ?>"
-                           value="<?= old('email', auth()->user()->email ?? null) ?>" required />
+                    <input type="email" class="form-control" name="email" autocomplete="email" placeholder="<?= lang('Auth.email') ?>" value="<?= old('email', auth()->user()->email ?? null) ?>" required />
                 </div>
 
                 <div class="d-grid col-12 col-md-8 mx-auto m-3">
                     <button type="submit" class="btn btn-primary btn-block"><?= lang('Auth.send') ?></button>
                 </div>
 
             </form>
+
+            <p class="text-center"><a href="<?= url_to('login') ?>"><?= lang('Auth.backToLogin') ?></a></p>
         </div>
     </div>
 </div>
 
-<?= $this->endSection() ?>
+<?= $this->endSection() ?>
```

Comment:
Please add a line break.

Reply:
Like this? It looks the same.

Reply:
Original
```php
        </div>
    </div>
</div>

<?= $this->endSection() ?>
```

Change by @kenjis
```php
        </div>
    </div>
</div>


<?= $this->endSection() ?>
```

two line breaks?

Reply:
No, at the end of the file.

`<?= $this->endSection() ?>\n`

Reply:
@jlopes90 I recommend you use `.editorconfig` aware IDE.
See https://editorconfig.org/